Azure/azure-cli
# [Storage] `az storage account create/update`: Enable account level immutability

**Description**

Enable account level immutability. When enabled, adds object level immutability for all the containers in the account by default.

**Testing Guide**

az storage account create -n {storage-account-name} -g {resource-group-name} --enable-vlw --immutability-period 10 --immutability-state Unlocked --allow-protected-append-writes

az storage account update -n {storage-account-name} --immutability-period 5 --immutability-state Disabled --allow-protected-append-writes false

---

This checklist is used to make sure that common guidelines for a pull request are followed.
